benchmarks/ps-experiment/llama-8b/ps_stack.yaml

```diff
@@ -0,0 +1,313 @@
+            - aibrix_download
+            - --model-uri
+            - tos://aibrix-artifact-testing/models/llama-3.1-8b-instruct/
+            - --local-dir
+            - /models/
+          env:
+            - name: DOWNLOADER_NUM_THREADS
+              value: "16"
+            - name: DOWNLOADER_ALLOW_FILE_SUFFIX
+              value: json, safetensors
+            - name: TOS_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: TOS_ACCESS_KEY
+                  name: tos-credential
+            - name: TOS_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: TOS_SECRET_KEY
+                  name: tos-credential
+            - name: TOS_ENDPOINT
+              value: https://tos-s3-cn-beijing.ivolces.com
+            - name: TOS_REGION
+              value: cn-beijing
+          image: aibrix-container-registry-cn-beijing.cr.volces.com/aibrix/runtime:v0.2.1
+          name: init-model
+          volumeMounts:
+            - mountPath: /models
+              name: model-hostpath
+      containers:
+        - name: "vllm"
+          #image: "lmcache/vllm-openai:2025-03-10"
+          image: aibrix-container-registry-cn-beijing.cr.volces.com/lmcache/vllm-openai:latest
+          command:
+          - "vllm"
+          - "serve"
+          - /models/llama-3.1-8b-instruct/
+          - --served-model-name
+          - "meta-llama/Llama-3.1-8B-Instruct"
+          - --trust-remote-code
+          - "--host"
+          - "0.0.0.0"
+          - "--port"
+          - "8000"
+          - "--enable-chunked-prefill"
+          - "false"
+          - "--max-model-len"
+          - "32000"
+          - "--dtype"
+          - "bfloat16"
+          - "--disable-log-requests"
+          - "--swap-space"
+          - "0"
+          - "--kv-transfer-config"
+          - '{"kv_connector":"LMCacheConnector","kv_role":"kv_both"}'
+          securityContext:
+            runAsNonRoot: false
+          imagePullPolicy: IfNotPresent
+          env:
+          - name: LMCACHE_USE_EXPERIMENTAL
+            value: "True"
+          - name: VLLM_RPC_TIMEOUT
+            value: "1000000"
+          - name: LMCACHE_LOCAL_CPU
+            value: "True"
+          - name: LMCACHE_MAX_LOCAL_CPU_SIZE
+            value: "120"
+          ports:
+            - name: "container-port"
+              containerPort: 8000          
+          startupProbe:
+            failureThreshold: 60
+            httpGet:
+              path: /health
+              port: 8000
+            initialDelaySeconds: 15
+            periodSeconds: 10
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /health
+              port: 8000
+            initialDelaySeconds: 15
+            periodSeconds: 10
+          resources:
+            requests:
+              memory: "150Gi"
+              cpu: "10"
+              nvidia.com/gpu: "1"
+            limits:
```

Comment:
cpu/mem limit is missing for ps_k8s_stack.yaml and ps_stack.yaml.

Reply:
I will keep this same at this moment. Just for reproduce purpose.